lua/lazyvim/plugins/extras/lang/twig.lua

```diff
@@ -0,0 +1,57 @@
+return {
+  recommended = {
+    ft = "twig",
+  },
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = { ensure_installed = { "twig" } },
+  },
+  {
+    "williamboman/mason.nvim",
+    opts = {
+      ensure_installed = {
+        "twiggy-language-server",
+        "twigcs",
+        "twig-cs-fixer",
+      },
+    },
+  },
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        ["twiggy_language_server"] = {
```

Comment:
```suggestion
        twiggy_language_server = {
```
should do it because it doesn't contain any invalid string characters to be enclosed in square brackets. Underscores are valid in strings.

Reply:
At first, I thought it was "twiggy-language-server" and not "twiggy_language_server", I forgot to remove the brackets afterwards. :smile: